### DISCUSSION TITLE
Need alert emails for Warning notifications from Glific

### DISCUSSION THREAD

**Discussion by gaurya95** [glific/glific#2932] [Ideas]
In an effort to improve our technical efficiency of the flows, I recently discovered that we get notifications from Glific. From what I understood in the thread - https://discord.com/channels/717975833226248303/1130805604458434580, the support for mails for critical notifications already exist. This feature request is to have mails for warning notifications as well.

**Comment by AkhileshNegi** [comment]
This is captured here
https://github.com/glific/glific/issues/2940
